### ISSUE TITLE
Very suprise to see this repo

### ISSUE THREAD

**Issue by Comdex** [wushilin/threads#1]
I am doing it,too....but I prefer to achieve more function to manage groutine like java concurrent package

**Comment by wushilin** [comment]
Yes, java tool set is more powerful because threads supporting more features, like terminating thread outside of the thread, agree this is still limited feature :)
